data/layout/telescope_positions-South-TestLayout.ecsv

```diff
@@ -1,26 +1,51 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string, description: common telescope name}
-# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST01 (central MST)}
-# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST01 (central MST)}
-# - {name: pos_z, unit: m, datatype: float32, description: telescope position (centre of sphere) upwards relative to NNN}
-# meta: !!omap
-# - data_type: positionfile
-# - description: "telescope positions for CTA South (Paranal)"
-# - center_lon: "-70.316345 deg"
-# - center_lat: "-24.683429 deg"
-# - center_alt: "2162.35 m"
-# - center_northing: "7269466.0 m"
-# - center_easting: "366822.0 m"
-# - corsika_obs_level: "2147 m"
-# - corsika_sphere_radius: {LST: 12.5 m, MST: 9.6 m, SST: 3 m}
-# - corsika_sphere_center: {LST: 16 m, MST: 9 m, SST: 3.25 m}
-# - EPSG: 32719
-# - comments: |
-#       version 0.1
-#       UTM Zone 19 (S)
-#       prod5
+# - {name: telescope_name, datatype: string}
+# - {name: pos_x, unit: m, datatype: float64}
+# - {name: pos_y, unit: m, datatype: float64}
+# - {name: pos_z, unit: m, datatype: float64}
+# meta:
+#   EPSG: 32719
+#   center_alt: !astropy.units.Quantity
+#     unit: &id001 !astropy.units.Unit {unit: m}
+#     value: 2147.0
```

Comment:
This should be 2162.35 m (from the values before the change). Notice that this is not the `corsika_obs_level` but the centre altitude.